onboarding.adoc

```diff
@@ -1,327 +1,86 @@
 
-Much of the focus on CI/CD has been on speeding up delivery of software to users.
-But these technologies can also be very helpful for providing contributors with "immediate gratification" by seeing their patches get merged and go out quickly.
-This is perhaps their most valuable aspect, as the pace of contributions is often determined by the pace of acceptance.
-If you accept faster, then you usually get new code faster.
+== Mentorship
 
-However, implementing CI/CD can be difficult for project leaders, because their automated testing isn't comprehensive, their builds aren't fully automated, and the consequences of letting a major bug get merged into the code can be very severe.
-Despite a good deal of lip service to the values and desirability of having a continuous code pipeline, irregular batch committer review is generally the reality, especially for smaller projects.
-Additionally, CI/CD software like Gerrit, Jenkins, and Travis can be difficult to set up in a way that supports projects' real releases.
 
-There is, however, one area where CI/CD is easier to implement—and most projects should be doing it right now.
 
-=== Automating documentation contributions
+== Resources: Onboarding examples from open source communities
 
-Documentation is one area in which most open source projects sorely need more contributions, and where acceptance of imperfect submissions carries relatively low risk.
-If your project is like most, you have a long list of things with obsolete or non-existent documentation, few documentation contributors, and little time for documentation review.
-And unless you have a large, mature, and heavily cross-referenced documentation set, the problems caused by accepting documentation errors are smaller than those caused by not having documentation on certain features at all.
-
-This means that, for most projects, self-serve CI/CD for documentation would be a real benefit.
-This is the reason why many projects adopt wikis for their documentation, since those allow for immediate editing and display.
-However, wikis make structuring documentation, rendering it in formats other than screen, or tying it to specific release versions difficult.
-For these reasons (and others), some projects have transitioned to building docs using CI/CD in DocBook, Sphinx, or AsciiDoc.
-
-== Mentoring new contributors
-
```

Comment:
Great work so far everyone! 

Quick question. Since we're adding this mentorship section back and expanding on it, thoughts on the edit process 🤔 
- Should I branch of this, make edits, and keep it consistent with changes going on here?
- Or just commit directly to this branch/PR? 

Let me know if have a preference, thanks!

Reply:
@rpaik is more experienced with GitHub than I am and may have a stronger opinion on workflow. I, however, would recommend/suggest working directly on this branch, so all three of us are hacking in the same space.

Reply:
Thanks for the thoughts! Didn't have sufficient permissions to push commits directly on this branch || existing PR, so opened this one on @rpaik 's fork to add back the previous/existing mentorship content (cc @rpaik )-- https://github.com/theopensourceway/guidebook/pull/176#discussion_r535819864 :D

Reply:
Gotcha. Thanks, @palante. @rpaik, what's your feeling on merging @palante's changes in your branch to return the prior mentoring content?